src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -1,19 +1,24 @@
+import { useFavorites } from 'context/favorites/FavoritesProvider';
 import { IconTypes } from 'enums/IconTypes';
 import { CitiesWeatherWidget } from 'components/CitiesWeatherWidget';
 import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
 import { FavoritesCitiesWeatherWidgetTestIds } from './FavoritesCitiesWeatherWidgetTestIds';
 
 const FavoritesCitiesWeatherWidget = () => {
-  const favoriteCities: string[] = [];
+  const { state } = useFavorites();
 
   return (
     <CitiesWeatherWidget data-testid={FavoritesCitiesWeatherWidgetTestIds.Container}>
-      <CitiesWeatherWidget.Title>favorites</CitiesWeatherWidget.Title>
-      <CitiesWeatherWidget.ShortcutsWrapper>
-        {favoriteCities.map((city) => (
-          <CityWeatherShortcut key={city} cityName={city} iconType={IconTypes.Heart} />
-        ))}
-      </CitiesWeatherWidget.ShortcutsWrapper>
+      {state.favorites.length > 0 && (
```

Comment:
**suggestion** i think we should stay consistent with our `renderContent` function